apps/mark/frontend/src/app_root.tsx

```diff
@@ -632,28 +598,16 @@ export function AppRoot({
   const machineConfig = machineConfigQuery.data;
 
   if (!cardReader) {
-    return (
-      <SetupCardReaderPage
-        useEffectToggleLargeDisplay={useEffectToggleLargeDisplay}
```

Comment:
@kofi-q could you remind me what the `useEffectToggleLargeDisplay` props were originally intended to do, and why they're no longer necessary?

Reply:
It was adjusting the font size to the largest on some of the simple full-page screens that just have a large icon and a prompt, to make the text stand out larger. Not needed anymore, with the current heading size hierarchy we have (and in the rare case we ever need to override the size mode, we can now wrap the relevant UI in a `ThemeProvider` instead)